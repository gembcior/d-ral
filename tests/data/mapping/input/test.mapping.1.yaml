field:
  name: test_name_1
