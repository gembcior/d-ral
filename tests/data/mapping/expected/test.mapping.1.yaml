field:
  default:
    name: test_name_1
  simple:
    name: test_name_1
